File: solvers.julia/parallel_robot_66/test_num_parallel_robot_66.jl
include("../_common/header.jl")
include("inidata_num_parallel_robot_66.jl")
include("coefs_parallel_robot_66.jl")
include("red_293x362_parallel_robot_66.jl")

function test_num_parallel_robot_66()

    Random.seed!(23)
    N = 10000
    d = 40

    stats = ini_stats("parallel_robot_66")

    for i in 1:N
        data = inidata_num_parallel_robot_66()
        C,U = coefs_parallel_robot_66(data)

        toc = @elapsed S = red_293x362_parallel_robot_66(C,data[3][1])
        stats.tm = [stats.tm; toc]
        if isempty(S); continue; end

        stats = update_stats(stats,S,C,U,d)
    end

    disp_stats(stats)

end

test_num_parallel_robot_66()

File: solvers.python/py_parallel_robot_66/test_num_parallel_robot_66.py
from numpy.random import seed
from time import time
from _common import ini_stats, update_stats, disp_stats
from py_parallel_robot_66 import inidata_num_parallel_robot_66, coefs_parallel_robot_66, red_293x362_parallel_robot_66

def test_num_parallel_robot_66():
    seed(23)
    N = 1000
    d = 40

    stats = ini_stats('parallel_robot_66')

    for i in range(N):
        data = inidata_num_parallel_robot_66()
        C,U = coefs_parallel_robot_66(data)

        tic = time()
        S = red_293x362_parallel_robot_66(C,data[2][0,0])
        stats['tm'].append(time()-tic)
        if S is None:
            continue

        stats = update_stats(stats,S,C,U,d)

    disp_stats(stats)